components/daisy/Dropdown.tsx

```diff
@@ -0,0 +1,28 @@
+export interface Props {
+  label: string;
+  items: Item[];
+}
+
+interface Item {
+  text: string;
+  href: string;
+}
+
+export default function Dropdown(props: Props) {
+  const { label, items } = props;
+
+  return (
+    <details className="dropdown">
+      <summary className="m-1 btn">{label}</summary>
+      <ul className="p-2 shadow menu dropdown-content z-[1] bg-base-100 rounded-box w-52">
```

Comment:
the z-[1] may be a problem if this component is rendered inside another that has a higher `z-index` value